node/hyperbahn/egress-nodes.js

```diff
@@ -71,15 +71,7 @@ EgressNodes.prototype.exitsFor = function exitsFor(serviceName) {
     var exitNodes = Object.create(null);
     for (var i = 0; i < k; i++) {
         var shardKey = serviceName + '~' + i;
-
-        // TODO ringpop will return itself if it cannot find
```

Comment:
It is. When we get a network partition and ringpop converges to be a ring of 1 I want to get an aggressive warning.

Where as being in the bootstrapping phase is normal operation during restarts and redeploys; network partitions are not normal and 90% of the time they are a bug in ringpop instead of a true network partition.